README.md

```diff
@@ -1,19 +1,17 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Portfolio
 
-> Description the project.
+> This is my portfolio repository. You will find here all my achievement and I can do for your organisation see you.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
 
 
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT**: I suggest you remove all unrelated sections of the project in your README and fill out the related(eg: `Getting Started: adding those commands you mentioned to clone the repo, Prerequisites tools,...)` sections to make your documentation professional. You can use **[this template](https://github.com/tresorsawasawa/MyPortfolio/blob/develop/README.md)** to help you correctly complete the related sections.